pkgs/applications/audio/LazyLimiter/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, faust2jack, faust2lv2 }:
+stdenv.mkDerivation rec {
+  name = "LazyLimiter-${version}";
+  version = "v0.3.01";
```

Comment:
Better move the "v" to the rev attribute of `fetchFromGitHub`, as versions should start with a digit. See http://nixos.org/nixpkgs/manual/#idm140737319248160